pyproject.toml

```diff
@@ -6,9 +6,10 @@ build-backend = "setuptools.build_meta"
 name = "arches_lingo"
 readme = "README.md"
 authors = [
-    {name = "Farallon Geographics, Inc", email = "dev@fargeo.com"}
+    {"name" = "Arches Project"}
 ]
-license = {text = "GNU AGPL3"}
+license = {text = "AGPL-3.0-or-later"}
+description = "An Arches application designed to support the creation of thesaurus/vocabulary data."
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Programming Language :: Python",
```

Comment:
I think we can remove the license classifier too, here's the warning I got doing the last 0.1.0 release of querysets:
```
        ********************************************************************************
        Please consider removing the following classifiers in favor of a SPDX license expression:

        License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)

        See https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license for details.
        ********************************************************************************
```